packages/mdl-button/index.scss

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-button {
```

Comment:
> I was planning on handling the transition when doing the shadows properly after elevation is added. Since I also need to handle it for desktop not having elevation by default with that.

I'd say add it now. We can just hard-code the values and then refactor once #4592 lands.

> Disabled raised button backgrounds do change

ah, you're right, sorry. Don't know how I missed that -_- ...

> We can't rely on ripple (JS) though, so the active state modifier is still extremely valuable.

I agree that needing javascript for a button is suboptimal. I still think the active state ux could be improved though. Perhaps something like what's shown in https://material-design.storage.googleapis.com/publish/material_v_8/material_ext_publish/0B3T7oTWa3HiFSTNhRndpWXBwRDA/components-buttons-flatbuttons.webm ?

Looking just at the "pressed" state within that video, the `:focus` state could be the background fade, and the `:active` state could be the radial expansion.